src/server/scripts/Northrend/VioletHold/boss_ichoron.cpp

```diff
@@ -335,54 +441,72 @@ class npc_ichor_globule : public CreatureScript
     {
         npc_ichor_globuleAI(Creature* creature) : ScriptedAI(creature)
         {
-            Initialize();
```

Comment:
by removing the call to Initialize() you left all the class fields uninitialized, triggering static analysis issues